controllers/v1/userOverviewController.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * name : userOverviewController.js
+ * author : Saish
+ * created-date : 20-june-2024
+ * Description : User overview api
+ */
+
+// Dependencies
+const observationsHelper = require(MODULES_BASE_PATH + "/observations/helper")
+const observationSubmissionsHelper = require(MODULES_BASE_PATH +
+    "/observationSubmissions/helper");
```

Comment:
dont create new controller file use existing one @borkarsaish65

Reply:
Did the change @ankit1shahu